blueprints/ember-timetree/index.js

```diff
@@ -8,7 +8,6 @@ module.exports = {
   },
 
   afterInstall: function() {
-    var json = require('../../../../bower.json');
-    return this.addBowerPackageToProject('d3', json.version);
+    return this.addBowerPackageToProject('d3');
```

Comment:
? This is so the `ember-timetree` npm package installed always installs the same version of `d3` we test and assume against. Otherwise it will default to what ever the latest version is, which could (theoretically) break timetree.